tools/python_integration_tests/classes/test.py

```diff
@@ -0,0 +1,179 @@
+# Copyright 2024 "Google LLC"
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import subprocess
+import time
+from collections import defaultdict
+import argparse
+from classes.ssh import SSHManager
+from classes.deployment import Deployment
+
+def run_command(cmd: str, err_msg: str = None) -> subprocess.CompletedProcess:
+        res = subprocess.run(cmd, shell=True, universal_newlines=True, check=True,
+                         stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        if res.returncode != 0:
+            raise subprocess.SubprocessError(f"{err_msg}:\n{res.stderr}")
+        return res
+
+class Test:
```

Comment:
Did you consider to inherit this class from `unittest.TestCase` and put deployment sutup/destroy into `setUp/tearDown` methods ?

Reply:
Updated to inherit `unittest.TestCase`